test/Microsoft.DotNet.Docker.Tests/ImageTests.cs

```diff
@@ -269,5 +266,24 @@ private static string GetIdentifier(string version, string type)
         {
             return $"{version}-{type}-{DateTime.Now.ToFileTime()}";
         }
+
+        private static string GetRid(ImageDescriptor imageDescriptor)
+        {
+            string rid;
```

Comment:
If we go by most common condition first, then would the following code be better?

```C#
string rid = "debian.8-x64";
if (imageDescriptor.IsArm)
{
	rid = "linux-arm";
}
else (string.Equals(imageDescriptor.OsVariant, "alpine", StringComparison.OrdinalIgnoreCase))
{
	rid = "alpine.3.6-x64";
}

return rid;
```

Reply:
Leaving as it - I prefer to not initialize variables like this.  I prefer the else pattern in general as it allows the compiler warning to catch any code paths that may have mistakenly left the variable undefined.